src/main/java/ch/njol/skript/effects/EffPotion.java

```diff
@@ -35,144 +29,70 @@
 import ch.njol.skript.util.PotionEffectUtils;
 import ch.njol.skript.util.Timespan;
 import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.bukkit.potion.PotionEffect;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Potion Effects")
-@Description("Apply or remove potion effects to/from entities.")
-@Examples({"apply swiftness 2 to the player",
-		"remove haste from the victim",
+@Description("Apply potion effects to/from entities.")
+@Examples({
+		"apply swiftness 2 to the player",
 		"on join:",
 		"\tapply potion of strength of tier {strength.%player%} to the player for 999 days",
```

Comment:
unrelated but can we change this to a list variable while we're editing this class?
```suggestion
		"\tapply potion of strength of tier {strength::%player's uuid%} to the player for 999 days",
```